pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -37,17 +43,39 @@ stdenv.mkDerivation rec {
     pythonPackages.python
   ] ++ lib.optional withRuby ruby;
 
+  patches = [
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/void-linux/void-packages/master/srcpkgs/notmuch/patches/0001-configure-detect-version-by-compiler-for-build.patch";
```

Comment:
Please use a commit id in the url, not just `master`

Reply:
Yikes.

Reply:
f95b944252e3a1fc900a18c8eeed59e6de9b6e9a